src/styles/index.scss

```diff
@@ -1,3 +1,97 @@
+$color-title: #060b35;
+$color-code: #616070;
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: red;
+}
+
+.card {
+  display: flex;
+  min-height: 100vh;
+  justify-content: center;
+  align-items: center;
+  background-color: blue;
+
+  &__block {
+    width: 200px;
+    height: 408px;
+    background-color: white;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 33px;
+
+    &--margin {
+      margin: 28px;
+    }
+  }
+}
+
+.photo {
+  width: 160px;
+  height: 134px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 40px 17px 17px;
+
+  &__title {
+    color: $color-title;
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__code {
+    color: $color-code;
+    font-family: "Roboto", sans-serif;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
```

Comment:
400 is a default value for the font-family, so it's redundant 
check other places